doc/developer_guide/psyir_symbols.rst

```diff
@@ -220,6 +216,41 @@ the `__contains__` method has no mechanism to pass a `scope_limit`
 optional argument. This would probably require a separate `setter` and
 `getter` to specify whether to check ancestors or not.
 
+Copying Symbols and Symbol Tables
+=================================
+
+Since Symbols can contain PSyIR nodes and other Symbols (e.g. as part
+of the definition of their precision or initial value), creating copies
+is not entirely straightforward. Every `Symbol` has the `copy` method:
+
+.. automethod:: psyclone.psyir.symbols.Symbol.copy
+
+This ensures that the precision and initial-value properties are copied
+appropriately (e.g. new PSyIR nodes constructed) but any Symbols
+referred to by those properties remain unchanged.
+
+When performing a deep copy of a PSyIR tree, obviously all Symbols will
```

Comment:
Maybe: "
This ensures that the precision and initial-value PSyIR subtrees are also copied appropriately, but any Symbols inside those nodes remain referring to the same Symbol objects (and therefore can still be used in the same scope).

However, when performing a deep copy of a PSyIR tree, all Symbols ..."

Reply:
I've re-worded. See what you think.